service/test/y2storage/agama_proposal_test.rb

```diff
@@ -24,6 +24,51 @@
 require "agama/storage/config"
 require "y2storage/agama_proposal"
 
+# @param config [Agama::Storage::Configs::Drive, Agama::Storage::Configs::Partition]
```

Comment:
These methods look like a hint that we need (or will shortly need) shared helpers or factories for creating config objects and similar stuff.

Reply:
Totally agree.